generate-dev-env.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+set -e
+
+# This script generates a dev env for developers
+# How to use: ./generate-dev-env.sh $CEPH_RELEASE $DISTRO $DISTRO_VERSION
+
+# VARIABLES
+BASEDIR=$(dirname "$0")
+
+# FUNCTIONS
+function echo_info {
+  echo ""
+  echo "*****************************************************************************************"
+  echo "$1"
+  echo "*****************************************************************************************"
+}
+
+function test_args {
+  if [ $# -ne 3 ]; then
+    echo_info "Please run the script like this: ./generate-dev-env.sh CEPH_RELEASE DISTRO DISTRO_VERSION"
+    exit 1
+  fi
+}
+
+function generate_dev_dir {
+  echo_info "Generating $BASEDIR/dev/$1/$2/$3"
+  mkdir -p $BASEDIR/dev/$1/$2/$3
+}
+
+function copy_files {
+  cp -Lrv ceph-releases/$1/$2/$3/* $BASEDIR/dev/$1/$2/$3
+}
+
+function wrong_combinaison {
+  echo_info "Incompatible combinaison of of CEPH_RELEASE DISTRO DISTRO_VERSION"
+  exit 1
+}
+
+# MAIN
+
+test_args $@
+
+case "$1" in
+  hammer|infernalis|jewel)
+    case "$2" in
+      centos)
+        case "$3" in
+          7)
+            generate_dev_dir $@
+            copy_files $@
+            ;;
+          *)
+            wrong_combinaison
+        esac
+        ;;
+      ubuntu)
+        case "$3" in
+          14.04|16.04)
+            generate_dev_dir $@
+            copy_files $@
+            ;;
+          *)
+            wrong_combinaison
+        esac
+        ;;
+      fedora)
+        case "$3" in
+          23)
+            generate_dev_dir $@
+            copy_files $@
+            ;;
+          *)
+            wrong_combinaison
```

Comment:
Is there a reason to hard-enforce the existing distro's / versions, rather than just doing the copy?  This is for developers, after all.